### PR TITLE
LPS-162762 Failed to upload file with multipart request in GraphQL

### PR DESCRIPTION
**Pull request by ccorreagg**
This PR contains the code to fix this bug: https://issues.liferay.com/browse/LPS-162762

Before the fix:

The response is:

```
{"errors":[{"message":"Validation error of type UnknownType: Unknown type Upload","locations":[],"errorType":"DataFetchingException","path":null,"extensions":{"exception":{"errno":400},"code":"Bad Request"}}],"data":null}
```

After the fix:

```
{
  "data": {
    "createSiteDocument": {
      "id": 43512,
      "title": "tarea.txt"
    }
  }
}
```

### CODE REVIEW

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:sf

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/multipart/MultipartUtil.java [hunk 1]
```diff
@@ -48,7 +46,9 @@ public static String getFileName(Part part) {
 	public static boolean isMultipartBody(Parameter parameter) {
 		Class<?> clazz = parameter.getType();
 
-		return StringUtil.contains(clazz.getTypeName(), "MultipartBody");
+		String typeName = clazz.getTypeName();
```
Comment (4lejandrito) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/multipart/MultipartUtil.java:49]: What is this fixing? Is StringUtil broken?

Comment (ccorreagg) [reply modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/multipart/MultipartUtil.java:49]: It has into account a separator, by default a comma (,), to check if the substring it is contained between separators. Initially, the code was without using the StringUtil, and I changed, but I didn't realize about the current functionality of StringUtils.contains, so I just left as I was before, and I found it easier to understand that with a separator that we don't need

Comment (4lejandrito) [reply modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/multipart/MultipartUtil.java:49]: Makes sense 👍

**Comment by ccorreagg** [comment]
ci:forward

**Pull request closed without merging**